Markdown renderer can't handle multiple-line code samples in lists.

The source that produced this is [here](https://github.com/GoogleChrome/workbox-microsite/blob/docs/src/get-started/gulp.md).

![rendered](https://cloud.githubusercontent.com/assets/8204171/25981015/6b77df0a-3687-11e7-9be0-89e80b0cd687.png)

This is malformed markdown.

To have something considered as part of the list item you need to indent by 4 spaces. Doing this fixes your problem.

Also worth calling out there is a mix of tabs and spaces that can upset all of this as well.

The following works:

```
---
layout: page
title: Get Started > Gulp
---

# Gulp

1. [Install Node.js](https://nodejs.org/en/).
1. Install the module with NPM.

    ```
    npm install --save-dev sw-build
    ```

1. Require `sw-build` in your build script.

    ```
    const swBuild = require('sw-build');
    ```

1. Add the following to your `gulpfile.js`:

    ```
    gulp.task('build', () => {
      const swBuild = require('sw-build');
      return swBuild.generateSW({
        rootDirectory: './build/',
        swDest: './build/sw.js',
        globPatterns: ['**\/*.{html,js,css}'],
        globIgnores: ['admin.html'],
        templatedUrls: {
          '/shell': ['shell.hbs', 'main.css', 'shell.css'],
        },
      })
      .then(() => {
        console.log('Service worker generated.');
      })
      .catch((err) => {
        console.log('[ERROR] This happened: ' + err);
      });
    })
    ```
```

--- a/src/get-started/gulp.md
+++ b/src/get-started/gulp.md
@@ -8,35 +8,35 @@ title: Get Started > Gulp
 1. [Install Node.js](https://nodejs.org/en/).
 1. Install the module with NPM.
 
-   ```
-   npm install --save-dev sw-build
-   ```
+    ```
+    npm install --save-dev sw-build
+    ```
 
 1. Require `sw-build` in your build script.
 
-   ```
-   const swBuild = require('sw-build');
-   ```
+    ```
+    const swBuild = require('sw-build');
+    ```
 
 1. Add the following to your `gulpfile.js`:
 
-	```
-	gulp.task('build', () => {
-	  const swBuild = require('sw-build');
-	  return swBuild.generateSW({
-	    rootDirectory: './build/',
-	    swDest: './build/sw.js',
-	    globPatterns: ['**\/*.{html,js,css}'],
-	    globIgnores: ['admin.html'],
-	    templatedUrls: {
-	      '/shell': ['shell.hbs', 'main.css', 'shell.css'],
-	    },
-	  })
-	  .then(() => {
-	    console.log('Service worker generated.');
-	  })
-	  .catch((err) => {
-	    console.log('[ERROR] This happened: ' + err);
-	  });
-	})
-	```
+    ```
+    gulp.task('build', () => {
+      const swBuild = require('sw-build');
+      return swBuild.generateSW({
+        rootDirectory: './build/',
+        swDest: './build/sw.js',
+        globPatterns: ['**\/*.{html,js,css}'],
+        globIgnores: ['admin.html'],
+        templatedUrls: {
+          '/shell': ['shell.hbs', 'main.css', 'shell.css'],
+        },
+      })
+      .then(() => {
+        console.log('Service worker generated.');
+      })
+      .catch((err) => {
+        console.log('[ERROR] This happened: ' + err);
+      });
+    })
+    ```

